openslides_backend/action/actions/mediafile/mixins.py

```diff
@@ -21,6 +21,9 @@ class MediafileMixin(Action):
     Overwrite update_instance(), check_permissions() and get_meeting_id().
     """
 
+    meeting_fields: list[str] = ["meeting_id", "access_group_ids"]
+    is_delete_action: bool = False
```

Comment:
Are the type annotations necessary? In the delete action they are omitted.

Reply:
because the delete action expands on the mixin. So the type annotation is already there